pkg/syncer/syncer.go

```diff
@@ -41,6 +41,11 @@ import (
 const resyncPeriod = 10 * time.Hour
 const SyncerNamespaceKey = "SYNCER_NAMESPACE"
 
+type Direction string
+
+const KcpToPcluster Direction = "kcpToPcluster"
+const PclusterToKcp Direction = "pclusterToKcp"
```

Comment:
Please add godoc.

I would also consider naming the constant PhysicalClusterToKCP.